presto-geospatial-toolkit/src/main/java/com/facebook/presto/geospatial/GeometrySerde.java

```diff
@@ -97,6 +103,26 @@ private static void writeSimpleGeometry(DynamicSliceOutput output, GeometryType
         output.appendBytes(shape);
     }
 
+    private static void writePoint(DynamicSliceOutput output, OGCGeometry geometry)
```

Comment:
Just to make sure, does this method have test coverage? I don't see any unit tests in this commit for this method.

Reply:
The tests for point serialization / deserialization were already there